from cleanstack.exceptions import ForbiddenError


class UserUnauthorizedError(ForbiddenError):
    pass
